plugin-build/src/main/kotlin/io/sentry/android/gradle/instrumentation/Instrumentable.kt

```diff
@@ -0,0 +1,21 @@
+package io.sentry.android.gradle.instrumentation
+
+import io.sentry.android.gradle.instrumentation.database.AndroidXSQLiteClassVisitor
+import org.objectweb.asm.ClassVisitor
+
+enum class Instrumentable(val fqName: String) {
```

Comment:
curious, why is it an enum?

Reply:
Hm, cause we need to do a `when` over it and it has a finite number of classes that we want to instrument.. I'm open for suggestions though, it could also be just a `Map<String, ClassVisitor>` I think, is that what you meant?

Reply:
yeah it will make more sense when we have more classes, wfm